serde-support/src/main/java/io/micronaut/serde/support/DefaultSerdeRegistry.java

```diff
@@ -165,12 +170,18 @@ public class DefaultSerdeRegistry implements SerdeRegistry {
      * @param conversionService The conversion service
      */
     public DefaultSerdeRegistry(
-        BeanContext beanContext,
-        ObjectSerializer objectSerializer,
-        ObjectDeserializer objectDeserializer,
-        Serde<Object[]> objectArraySerde,
-        SerdeIntrospections introspections,
-        ConversionService conversionService) {
+            BeanContext beanContext,
+            ObjectSerializer objectSerializer,
+            ObjectDeserializer objectDeserializer,
+            Serde<Object[]> objectArraySerde,
+            SerdeIntrospections introspections,
+            ConversionService conversionService,
+            SerdeConfiguration serdeConfiguration,
+            SerializationConfiguration serializationConfiguration,
+            DeserializationConfiguration deserializationConfiguration) {
+        this.serdeConfiguration = serdeConfiguration;
```

Comment:
I think this constructor is used in Micronaut Data

Reply:
oh didnt notice this class wasnt internal